flang/runtime/support.cpp

```diff
@@ -17,6 +18,27 @@ bool RTDEF(IsContiguous)(const Descriptor &descriptor) {
   return descriptor.IsContiguous();
 }
 
+void RTDEF(CopyAndUpdateDescriptor)(Descriptor &to, const Descriptor &from,
```

Comment:
Please check with Slava to find out which magic device macros need to be applied here.

Reply:
Yes, it is correct.